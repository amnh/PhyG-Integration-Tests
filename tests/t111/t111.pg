--build(distance, dwag, block), inputting fasta files with missing and chopped data
set(seed:1634561640)
read(fasta:"dec258_12Saln_missing.fas")
read(fasta:"dec258_16Saln_chopped.fasta")
build(distance, dWag, block, eun, displaytrees:3)
report("t111.csv", data, overwrite)
report("t111.tre", newick, overwrite)