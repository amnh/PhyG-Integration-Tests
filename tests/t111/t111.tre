(Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Aristeidae_Aristeomorpha_foliacea_NC_039153:326.0,((((Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Cambaridae_Procambarus_clarkii_NC_016926:140.0,((Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_laevis_KX343004:43.0,Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_validus_NC_041107:34.0)HTU32:75.0,(Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Potamoidea_Potamidae_Potamiscus_motuoensis_KY285013:88.0,(Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Portunoidea_Portunidae_Portunus_trituberculatus_NC_005037:114.0,((Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_argus_NC_039671:103.0,Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_cygnus_NC_028024:84.0)HTU35:96.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Alpheoidea_Alpheidae_Alpheus_distinguendus_NC_014883:153.0,Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Spongicola_levigatus_KU188325:88.0)HTU56:66.0)HTU52:44.0)HTU53:43.0)HTU51:31.0)HTU50:24.0)HTU49:33.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Crangonoidea_Crangonidae_Crangon_hakodatei_KU641481:141.0,Eucarida_Decapoda_Pleocyemata_Caridea_Nematocarcinoidea_Rhynchocinetidae_Rhynchocinetes_durbanensis_NC_029372:197.0)HTU55:66.0)HTU54:41.0,(Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Sergestidae_Sergia_lucens_NC_037043:93.0,(((Eucarida_Decapoda_Pleocyemata_Stenopodidea_Stenopodidae_Stenopus_hispidus_NC_018097:124.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Bresilioidea_Alvinocarididae_Opaepele_loihi_NC_020311:136.0,Eucarida_Euphausiacea_Euphausiidae_Euphausia_pacifica_NC_016184:108.0)HTU46:48.0)HTU45:29.0,(Eucarida_Decapoda_Pleocyemata_Anomura_Paguroidea_Diogenidae_Clibanarius_infraspinatus_NC_025776:137.0,(Eucarida_Decapoda_Pleocyemata_Anomura_Galatheoidea_Porcellanidae_Petrolisthes_haswelli_NC_025572:144.0,Eucarida_Decapoda_Pleocyemata_Axiidea_Callianassidae_Nihonotrypaea_thermophila_NC_019610:157.0)HTU48:48.0)HTU47:43.0)HTU44:32.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Palaemonoidea_Palaemonidae_Hymenocera_picta_NC_039631:164.0,(Eucarida_Decapoda_Pleocyemata_Brachyura_Dromiacea_Homoloidea_Homolidae_Homologenus_malayensis_NC_026080:114.0,Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Austinogebia_edulis_NC_019606:112.0)HTU58:43.0)HTU57:34.0)HTU43:29.0)HTU60:26.0)HTU42:37.0,((Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_torrentium_NC_033504:67.0,(Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_pallipes_NC_026560:68.0,(Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Bythograeoidea_Bythograeidae_Austinograea_rodriguezensis_NC_020312:135.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Atyoidea_Atyidae_Caridina_gracilipes_NC_024751:200.0,(Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Globospongicola_spinulatus_KU188326:123.0,(Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Luciferidae_Belzebub_intermedius_NC_039641:82.0,Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Gebiacantha_plantae_NC_041152:82.0)HTU39:27.0)HTU38:53.0)HTU37:49.0)HTU36:64.0)HTU34:20.0)HTU33:103.0,(Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Pleoticus_muelleri_NC_039964:58.0,Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Solenocera_crassicornis_NC_030280:72.0)HTU41:40.0)HTU59:17.0)HTU40:169.0)HTU31[12131.0];
(Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Aristeidae_Aristeomorpha_foliacea_NC_039153:290.0,(((Eucarida_Decapoda_Pleocyemata_Brachyura_Dromiacea_Homoloidea_Homolidae_Homologenus_malayensis_NC_026080:119.0,(Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Bythograeoidea_Bythograeidae_Austinograea_rodriguezensis_NC_020312:126.0,(Eucarida_Decapoda_Pleocyemata_Anomura_Paguroidea_Diogenidae_Clibanarius_infraspinatus_NC_025776:136.0,Eucarida_Decapoda_Pleocyemata_Caridea_Atyoidea_Atyidae_Caridina_gracilipes_NC_024751:170.0)HTU54:55.0)HTU53:43.0)HTU52:49.0,(Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Sergestidae_Sergia_lucens_NC_037043:100.0,((Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Potamoidea_Potamidae_Potamiscus_motuoensis_KY285013:159.0,(Eucarida_Euphausiacea_Euphausiidae_Euphausia_pacifica_NC_016184:113.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Alpheoidea_Alpheidae_Alpheus_distinguendus_NC_014883:230.0,Eucarida_Decapoda_Pleocyemata_Caridea_Bresilioidea_Alvinocarididae_Opaepele_loihi_NC_020311:114.0)HTU51:101.0)HTU50:49.0)HTU49:34.0,(Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Cambaridae_Procambarus_clarkii_NC_016926:149.0,((Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_laevis_KX343004:41.0,Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_validus_NC_041107:37.0)HTU32:62.0,(Eucarida_Decapoda_Pleocyemata_Anomura_Galatheoidea_Porcellanidae_Petrolisthes_haswelli_NC_025572:103.0,(Eucarida_Decapoda_Pleocyemata_Axiidea_Callianassidae_Nihonotrypaea_thermophila_NC_019610:84.0,Eucarida_Decapoda_Pleocyemata_Caridea_Crangonoidea_Crangonidae_Crangon_hakodatei_KU641481:100.0)HTU58:42.0)HTU57:31.0)HTU56:35.0)HTU55:38.0)HTU48:43.0)HTU60:26.0)HTU47:53.0,(Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Solenocera_crassicornis_NC_030280:82.0,(Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Pleoticus_muelleri_NC_039964:66.0,((Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Austinogebia_edulis_NC_019606:117.0,(Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_torrentium_NC_033504:57.0,(Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_pallipes_NC_026560:61.0,(Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Portunoidea_Portunidae_Portunus_trituberculatus_NC_005037:148.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Palaemonoidea_Palaemonidae_Hymenocera_picta_NC_039631:153.0,(Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Globospongicola_spinulatus_KU188326:100.0,(Eucarida_Decapoda_Pleocyemata_Stenopodidea_Stenopodidae_Stenopus_hispidus_NC_018097:104.0,(Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Spongicola_levigatus_KU188325:112.0,(Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Luciferidae_Belzebub_intermedius_NC_039641:179.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Nematocarcinoidea_Rhynchocinetidae_Rhynchocinetes_durbanensis_NC_029372:92.0,Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Gebiacantha_plantae_NC_041152:73.0)HTU45:28.0)HTU44:136.0)HTU43:52.0)HTU42:52.0)HTU41:87.0)HTU40:76.0)HTU39:81.0)HTU36:18.0)HTU35:79.0)HTU34:58.0,(Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_argus_NC_039671:118.0,Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_cygnus_NC_028024:120.0)HTU38:100.0)HTU37:81.0)HTU33:28.0)HTU59:51.0)HTU46:158.0)HTU31[11977.0];
(Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Aristeidae_Aristeomorpha_foliacea_NC_039153:338.0,((((Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_pallipes_NC_026560:168.0,(Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Bythograeoidea_Bythograeidae_Austinograea_rodriguezensis_NC_020312:158.0,((Eucarida_Decapoda_Pleocyemata_Caridea_Nematocarcinoidea_Rhynchocinetidae_Rhynchocinetes_durbanensis_NC_029372:224.0,Eucarida_Decapoda_Pleocyemata_Caridea_Palaemonoidea_Palaemonidae_Hymenocera_picta_NC_039631:179.0)HTU57:50.0,(Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Gebiacantha_plantae_NC_041152:90.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Atyoidea_Atyidae_Caridina_gracilipes_NC_024751:97.0,Eucarida_Decapoda_Pleocyemata_Caridea_Bresilioidea_Alvinocarididae_Opaepele_loihi_NC_020311:93.0)HTU56:38.0)HTU58:21.0)HTU55:57.0)HTU54:60.0)HTU53:43.0,((Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_laevis_KX343004:42.0,Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_validus_NC_041107:37.0)HTU32:88.0,((Eucarida_Decapoda_Pleocyemata_Brachyura_Dromiacea_Homoloidea_Homolidae_Homologenus_malayensis_NC_026080:104.0,Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Potamoidea_Potamidae_Potamiscus_motuoensis_KY285013:83.0)HTU47:43.0,(Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Luciferidae_Belzebub_intermedius_NC_039641:114.0,Eucarida_Decapoda_Pleocyemata_Axiidea_Callianassidae_Nihonotrypaea_thermophila_NC_019610:94.0)HTU48:36.0)HTU46:18.0)HTU60:28.0)HTU44:49.0,(Eucarida_Euphausiacea_Euphausiidae_Euphausia_pacifica_NC_016184:121.0,(Eucarida_Decapoda_Pleocyemata_Anomura_Galatheoidea_Porcellanidae_Petrolisthes_haswelli_NC_025572:196.0,((Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_argus_NC_039671:138.0,Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_cygnus_NC_028024:118.0)HTU37:114.0,(Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Spongicola_levigatus_KU188325:118.0,(Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Globospongicola_spinulatus_KU188326:120.0,Eucarida_Decapoda_Pleocyemata_Stenopodidea_Stenopodidae_Stenopus_hispidus_NC_018097:125.0)HTU45:52.0)HTU51:135.0)HTU50:57.0)HTU52:47.0)HTU49:47.0)HTU43:60.0,((Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Sergestidae_Sergia_lucens_NC_037043:84.0,(Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Austinogebia_edulis_NC_019606:128.0,(Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Cambaridae_Procambarus_clarkii_NC_016926:97.0,(Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_torrentium_NC_033504:83.0,(Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Portunoidea_Portunidae_Portunus_trituberculatus_NC_005037:136.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Crangonoidea_Crangonidae_Crangon_hakodatei_KU641481:125.0,(Eucarida_Decapoda_Pleocyemata_Anomura_Paguroidea_Diogenidae_Clibanarius_infraspinatus_NC_025776:134.0,Eucarida_Decapoda_Pleocyemata_Caridea_Alpheoidea_Alpheidae_Alpheus_distinguendus_NC_014883:182.0)HTU40:61.0)HTU39:50.0)HTU38:59.0)HTU36:27.0)HTU35:55.0)HTU34:80.0)HTU33:38.0,(Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Pleoticus_muelleri_NC_039964:68.0,Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Solenocera_crassicornis_NC_030280:86.0)HTU42:45.0)HTU59:25.0)HTU41:186.0)HTU31[11856.0];
