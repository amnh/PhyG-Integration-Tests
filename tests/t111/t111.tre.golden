(Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Aristeidae_Aristeomorpha_foliacea_NC_039153:521.5,(Eucarida_Decapoda_Pleocyemata_Anomura_Galatheoidea_Porcellanidae_Petrolisthes_haswelli_NC_025572:511.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Bresilioidea_Alvinocarididae_Opaepele_loihi_NC_020311:343.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Alpheoidea_Alpheidae_Alpheus_distinguendus_NC_014883:428.0,(Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Bythograeoidea_Bythograeidae_Austinograea_rodriguezensis_NC_020312:382.0,((Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Pleoticus_muelleri_NC_039964:257.0,Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Solenocera_crassicornis_NC_030280:248.5)HTU35:158.5,((Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Sergestidae_Sergia_lucens_NC_037043:342.5,(Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Austinogebia_edulis_NC_019606:310.0,Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Gebiacantha_plantae_NC_041152:335.0)HTU38:198.5)HTU45:132.0,(((Eucarida_Euphausiacea_Euphausiidae_Euphausia_pacifica_NC_016184:278.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Atyoidea_Atyidae_Caridina_gracilipes_NC_024751:375.5,Eucarida_Decapoda_Pleocyemata_Caridea_Crangonoidea_Crangonidae_Crangon_hakodatei_KU641481:324.0)HTU60:251.5)HTU44:155.5,((Eucarida_Decapoda_Pleocyemata_Anomura_Paguroidea_Diogenidae_Clibanarius_infraspinatus_NC_025776:360.0,(Eucarida_Decapoda_Pleocyemata_Axiidea_Callianassidae_Nihonotrypaea_thermophila_NC_019610:387.0,Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Portunoidea_Portunidae_Portunus_trituberculatus_NC_005037:364.0)HTU40:170.0)HTU41:166.0,((Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Cambaridae_Procambarus_clarkii_NC_016926:235.0,(Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_pallipes_NC_026560:207.0,Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_torrentium_NC_033504:195.0)HTU32:101.0)HTU33:168.0,(Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Luciferidae_Belzebub_intermedius_NC_039641:400.5,Eucarida_Decapoda_Pleocyemata_Caridea_Palaemonoidea_Palaemonidae_Hymenocera_picta_NC_039631:406.0)HTU53:165.0)HTU52:163.0)HTU58:151.0)HTU34:163.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Nematocarcinoidea_Rhynchocinetidae_Rhynchocinetes_durbanensis_NC_029372:401.5,((Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_laevis_KX343004:255.5,Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_validus_NC_041107:213.0)HTU36:215.0,(Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Potamoidea_Potamidae_Potamiscus_motuoensis_KY285013:297.0,(Eucarida_Decapoda_Pleocyemata_Brachyura_Dromiacea_Homoloidea_Homolidae_Homologenus_malayensis_NC_026080:348.5,((Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_argus_NC_039671:372.5,Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_cygnus_NC_028024:356.5)HTU39:199.5,(Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Spongicola_levigatus_KU188325:314.5,(Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Globospongicola_spinulatus_KU188326:320.5,Eucarida_Decapoda_Pleocyemata_Stenopodidea_Stenopodidae_Stenopus_hispidus_NC_018097:321.5)HTU43:141.5)HTU50:249.0)HTU49:178.0)HTU48:189.5)HTU47:190.0)HTU46:191.0)HTU51:162.0)HTU59:196.5)HTU37:149.5)HTU42:176.0)HTU55:190.5)HTU56:168.5)HTU57:185.5)HTU54:415.0)HTU31[12033.0];
(Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Aristeidae_Aristeomorpha_foliacea_NC_039153:581.0,(Eucarida_Decapoda_Pleocyemata_Anomura_Galatheoidea_Porcellanidae_Petrolisthes_haswelli_NC_025572:427.5,(Eucarida_Decapoda_Pleocyemata_Caridea_Bresilioidea_Alvinocarididae_Opaepele_loihi_NC_020311:326.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Alpheoidea_Alpheidae_Alpheus_distinguendus_NC_014883:385.0,(Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Bythograeoidea_Bythograeidae_Austinograea_rodriguezensis_NC_020312:382.5,((Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Pleoticus_muelleri_NC_039964:234.0,Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Solenocera_crassicornis_NC_030280:240.0)HTU35:144.0,(Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Sergestidae_Sergia_lucens_NC_037043:298.5,(((Eucarida_Decapoda_Pleocyemata_Anomura_Paguroidea_Diogenidae_Clibanarius_infraspinatus_NC_025776:375.5,(Eucarida_Euphausiacea_Euphausiidae_Euphausia_pacifica_NC_016184:319.5,(Eucarida_Decapoda_Pleocyemata_Caridea_Palaemonoidea_Palaemonidae_Hymenocera_picta_NC_039631:369.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Atyoidea_Atyidae_Caridina_gracilipes_NC_024751:404.5,Eucarida_Decapoda_Pleocyemata_Caridea_Crangonoidea_Crangonidae_Crangon_hakodatei_KU641481:336.5)HTU60:250.5)HTU40:178.5)HTU44:157.0)HTU43:146.5,(Eucarida_Decapoda_Pleocyemata_Axiidea_Callianassidae_Nihonotrypaea_thermophila_NC_019610:350.0,((Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Cambaridae_Procambarus_clarkii_NC_016926:248.5,(Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_pallipes_NC_026560:223.5,Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_torrentium_NC_033504:203.5)HTU32:101.5)HTU33:176.5,(Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Portunoidea_Portunidae_Portunus_trituberculatus_NC_005037:375.0,((Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Austinogebia_edulis_NC_019606:338.0,Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Gebiacantha_plantae_NC_041152:362.0)HTU38:152.0,(Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Luciferidae_Belzebub_intermedius_NC_039641:479.0,(Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_argus_NC_039671:396.0,Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_cygnus_NC_028024:374.0)HTU39:208.5)HTU52:181.5)HTU53:182.5)HTU51:135.0)HTU50:138.5)HTU58:172.0)HTU34:144.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Nematocarcinoidea_Rhynchocinetidae_Rhynchocinetes_durbanensis_NC_029372:415.5,((Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_laevis_KX343004:264.5,Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_validus_NC_041107:222.0)HTU36:195.0,(Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Potamoidea_Potamidae_Potamiscus_motuoensis_KY285013:318.5,(Eucarida_Decapoda_Pleocyemata_Brachyura_Dromiacea_Homoloidea_Homolidae_Homologenus_malayensis_NC_026080:351.0,(Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Spongicola_levigatus_KU188325:339.0,(Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Globospongicola_spinulatus_KU188326:333.0,Eucarida_Decapoda_Pleocyemata_Stenopodidea_Stenopodidae_Stenopus_hispidus_NC_018097:334.5)HTU42:128.5)HTU48:187.0)HTU47:197.0)HTU46:217.0)HTU45:174.5)HTU49:194.0)HTU59:167.5)HTU37:167.0)HTU41:196.0)HTU55:220.0)HTU56:195.0)HTU57:170.5)HTU54:508.0)HTU31[11984.0];
(Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Aristeidae_Aristeomorpha_foliacea_NC_039153:488.5,(Eucarida_Decapoda_Pleocyemata_Anomura_Galatheoidea_Porcellanidae_Petrolisthes_haswelli_NC_025572:479.0,(Eucarida_Decapoda_Pleocyemata_Caridea_Bresilioidea_Alvinocarididae_Opaepele_loihi_NC_020311:309.5,(Eucarida_Decapoda_Pleocyemata_Caridea_Alpheoidea_Alpheidae_Alpheus_distinguendus_NC_014883:390.0,(Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Bythograeoidea_Bythograeidae_Austinograea_rodriguezensis_NC_020312:342.0,((Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Pleoticus_muelleri_NC_039964:212.5,Eucarida_Decapoda_Dendrobranchiata_Penaeoidea_Solenoceridae_Solenocera_crassicornis_NC_030280:228.0)HTU35:161.5,(Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Sergestidae_Sergia_lucens_NC_037043:271.5,(((Eucarida_Decapoda_Pleocyemata_Anomura_Paguroidea_Diogenidae_Clibanarius_infraspinatus_NC_025776:357.0,((Eucarida_Decapoda_Pleocyemata_Caridea_Palaemonoidea_Palaemonidae_Hymenocera_picta_NC_039631:322.5,(Eucarida_Decapoda_Pleocyemata_Caridea_Atyoidea_Atyidae_Caridina_gracilipes_NC_024751:407.5,Eucarida_Decapoda_Pleocyemata_Caridea_Crangonoidea_Crangonidae_Crangon_hakodatei_KU641481:351.5)HTU60:160.0)HTU40:214.5,(Eucarida_Decapoda_Pleocyemata_Caridea_Nematocarcinoidea_Rhynchocinetidae_Rhynchocinetes_durbanensis_NC_029372:433.0,Eucarida_Euphausiacea_Euphausiidae_Euphausia_pacifica_NC_016184:350.0)HTU46:171.5)HTU45:184.5)HTU44:150.5,((Eucarida_Decapoda_Pleocyemata_Axiidea_Callianassidae_Nihonotrypaea_thermophila_NC_019610:378.5,Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Portunoidea_Portunidae_Portunus_trituberculatus_NC_005037:361.5)HTU41:173.5,((Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Cambaridae_Procambarus_clarkii_NC_016926:247.0,(Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_pallipes_NC_026560:223.5,Eucarida_Decapoda_Pleocyemata_Astacidea_Astacoidea_Astacidae_Austropotamobius_torrentium_NC_033504:196.5)HTU32:98.0)HTU33:167.5,((Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Austinogebia_edulis_NC_019606:318.0,Eucarida_Decapoda_Pleocyemata_Gebiidea_Upogebiidae_Gebiacantha_plantae_NC_041152:350.0)HTU38:140.0,(Eucarida_Decapoda_Dendrobranchiata_Sergestoidea_Luciferidae_Belzebub_intermedius_NC_039641:461.0,(Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_argus_NC_039671:392.5,Eucarida_Decapoda_Pleocyemata_Achelata_Palinuroidea_Palinuridae_Panulirus_cygnus_NC_028024:371.5)HTU39:216.5)HTU52:198.5)HTU53:147.5)HTU51:125.0)HTU58:158.0)HTU34:157.5,((Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_laevis_KX343004:222.0,Eucarida_Decapoda_Pleocyemata_Polychelida_Eryonoidea_Polychelidae_Pentacheles_validus_NC_041107:185.5)HTU36:234.5,(Eucarida_Decapoda_Pleocyemata_Brachyura_Eubrachyura_Potamoidea_Potamidae_Potamiscus_motuoensis_KY285013:286.0,(Eucarida_Decapoda_Pleocyemata_Brachyura_Dromiacea_Homoloidea_Homolidae_Homologenus_malayensis_NC_026080:323.0,(Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Spongicola_levigatus_KU188325:326.0,(Eucarida_Decapoda_Pleocyemata_Stenopodidea_Spongicolidae_Globospongicola_spinulatus_KU188326:320.5,Eucarida_Decapoda_Pleocyemata_Stenopodidea_Stenopodidae_Stenopus_hispidus_NC_018097:328.0)HTU43:134.0)HTU50:200.0)HTU49:203.5)HTU48:232.5)HTU47:165.5)HTU59:225.5)HTU37:142.0)HTU42:190.0)HTU55:245.5)HTU56:174.0)HTU57:201.0)HTU54:373.5)HTU31[11947.0];
